test/Zapper/vector-zapper-test.ts

```diff
@@ -0,0 +1,31 @@
+
+/***
+ *======== HOW TO USE THIS FILE ========
```

Comment:
So this test isn't working correct?

Reply:
we definitely need a zapper test for all tbh